chess/doc/index.rst

```diff
@@ -0,0 +1,74 @@
+
+* 1 Chess board
+* 2 Съеденные фигуры
+* 3 История ходов в возможностью просмотра ходов
+
+**New game**
+
+* 1 Click в "Шахматы" в главном меню сайта.
+
+  * 1.1 Create game, при этом режиме приглашение на игру отправиться случайному игроку зарегистрированного на сайте.
+  * 1.2 Play with a friend, при этом режиме можно выбирать кому будет отправление приглашение с игрой.
+
+* 2 выберете настройки игры
+
+  * 2.1 Укажите Game type. Standart - без учета времени. Blitz - с фиксированным временем на игру.  Limited time с фиксированным временем на ход. При выборе Blitz или Limited time открывается новое поле Time, в котором указывается время для игры в соответсвующих стилях. D - время в днях, H - время в часах, M - время в минутах, S - время в секундах. (по умолчанию указано время в 15 минут)
+  * 2.2 Выберете Figure color
+  * 2.3 Нажмите Create game. Игра начнется когда второй игрок примет приглашение.
+
+**Приглашение**
+
+* 1 Откройте в меню Chess.
+
+  * 1.1 Кликните New games. Отобразиться список приглашений в игру от других пользователей
+
+    * 1.1.1 Click in Accept откроется новая страница и начнется игра
+    * 1.1.2 Click in Refuse отказ от игры, запись переместиться в Refusals of games
+
+  * 1.2 Кликните Active games. Откроется список активных игр. Можно перейти к игре кликнув по кнопке Go to the game
+  * 1.3 Кликните Completed games список завершенных игр.
+  * 1.4 Кликните My invitations список приглашений, которые отправили вы. При нажатии на кнопку cancel отправленное приглашение отменяется и переноситься в список Canceled games
+  * 1.5 Кликните Refusals of games - список приглашений на игру, пользователи которых отказались принимать эту игру.
+  * 1.6 Кликните Canceled games - список отмененных приглашений в игру.
+
+**Game**
+
+* 1 Нажмите на левую кнопку мыши и удерживая его передвигайте фигуру на шахматной доске в нужное место.
+
+  * 1.1 Нажмите на появившийся PGN справа от шахматной доски, для просмотра сделанного хода.
+
+* 2 Нажмите на кнопку Surrender для того чтобы сдаться
+* 3 Нажмите на кнопку Sugest a draw для того чтобы придложить ничью другому игроку.
+* 4 Нажмите на кнопку setting для того чтобы просмотреть FEN и поменять ориентацию шахматной доски.
+* 5 Поставьте галочку на Window chat для того чтобы открыть чат.
+* 6 Напишите сообщение в чате и отправьте его нажав на кнопку send или комбинация клавишь ctrl + enter
+
+**Конец игры**
+
+Конец игры определяется правилами игры или возможными некоторыми действиями игроков такие как:
+
+* 1 Ничья после 50 ходов.
+* 2 Правило трех повторений.
+* 4 Автоничья, если недостаточно фигур для мата.
+* 5 Один из игроков сдался.
+* 6 Один из игроков принял ничью.
+* 7 У одного из игроков кончилось время.
+
+**Результат игры**
+
+Результатом игры является новый рейтинг игрока, который вычисляется по системе Elo, и записывается слева от шахматной доски.
+
+**Note.** To use the module, you need to be sure that your odoo instance support longpolling, i.e. Instant Messaging works.
```

Comment:
add link to this page https://odoo-development.readthedocs.io/en/latest/admin/longpolling.html